src/nwb_benchmarks/core/_nwb_helpers.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+import pynwb
+
+
+def get_object_by_name(nwbfile: pynwb.NWBFile, object_name: str) -> Any:
+    """Simple helper function to retrieve a neurodata object by its name, if it is unique."""
```

Comment:
I think it's ok to use this in `setup` methods for tests, but I think we should be careful when using this in benchmarks, since this function performs an iterative search. While this search should not require remote data access, it still could lead to additional time and memory that we may not want to capture.

Reply:
Yes, my intention for this is only for use in `setup` to help get an object to run an actual benchmark on

Added an extra bit to the docstring about that